docs/04-guides/get-viewer-engagement/_tabs.mdx

```diff
@@ -0,0 +1,26 @@
+import TabbedPages from "@site/src/components/TabbedPages";
+
+<head>
+  <title>Get viewer engagement | Livepeer Studio Docs</title>
+  <meta
+    property="og:title"
+    content="Get viewer engagement | Livepeer Studio Docs"
+  />
+</head>
+
+# Get viewer engagement
+
+This guide provides information about adding and retrieving viewership to a video using
+the [dashboard](./dashboard) or the [Livepeer player API](./api).
+
+## Caveats
+
+The viewership API is still in development. Currently, we only support start views,
+views when you open or start watching an asset. In the future, we'll support additional viewership and engagement metrics so you can understand your user's experience.  
```

Comment:
@ichung08 I committed this change directly, not sure if I'm supposed to do that 🙃 removed mention of "total views" being added later.